### PR TITLE
New blog on pulling images from Docker

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@sanjabonic @TomSweeneyRedHat PTAL

File: _posts/2018-09-25-pulling-images-from-docker.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+author: Dan Walsh
+categories: [blogs]
+tags: podman, containers
+---
+
+![podman logo](https://podman.io/images/podman.svg)
+
+# Cool things you can do with Podman.
+## Pushing content directly to Docker Daemon...
+
```
Comment (TomSweeneyRedHat) [inline _posts/2018-09-25-pulling-images-from-docker.md]: Did you want to add:
```
## By Dan Walsh @rhatdan
``` 
Or maybe three pounds?

Otherwise LGTM

Comment (rhatdan) [reply _posts/2018-09-25-pulling-images-from-docker.md]: Yup

**Comment by rhatdan** [comment]
@TomSweeneyRedHat Fixed it up.

**Comment by rhatdan** [comment]
Lets publish in the morning.

**Comment by TomSweeneyRedHat** [comment]
Ack, I'll push it out in the am.

**Pull request merged**